charts/tidb-operator/values.yaml

```diff
@@ -35,6 +35,7 @@ features: []
 # - AdvancedStatefulSet=false
 # - StableScheduling=true
 # - AutoScaling=false
+# - DynamicConfiguration=true
```

Comment:
We should control this behavior per TidbCluster instead of a global switch.

Reply:
make sense